### PR TITLE
feat(plugins/lines): add `plugin_lines_delay` to mitigate first query results

### LINKED ISSUE #1480: Lines plugin not working correctly
### Discussed in https://github.com/lowlighter/metrics/discussions/1445

<div type='discussions-op-text'>

<sup>Originally posted by **revilowaldow** May 16, 2023</sup>
The lines plugin is wrapping incorrectly and leaving large amounts of whitespace.
it used to render correctly.

### To reproduce:
Look at the example:
https://github.com/lowlighter/metrics/blob/examples/metrics.plugin.lines.history.svg
image

How it looks on my personal metrics:
image
https://github.com/revilowaldow/revilowaldow/blob/main/metrics.core.svg

How it used to look and how it should look:
image
https://github.com/revilowaldow/revilowaldow/commit/0ff1f2335e51d7e5ecfb82299e4676dc51b4b9a3
</div>

### PR DESCRIPTION
**Pull request by lowlighter**
Add a new option `plugin_lines_delay` to perform a second query on `rest.repos.getContributorsStats` (which is sometimes returning zeroed values on first query, but returning correct values upon subsequent queries after a certain amount of time) to mitigate the mentioned behaviour in parenthesis

Also fixes #1480 on the way

### CODE REVIEW

**Comment by revilowaldow** [comment]
What is a suggested delay to use?

**Comment by lowlighter** [comment]
It probably depends on how much diff you've written overall, but a few seconds seems to be more than enough from what I've seen.
Personally I intend to set this option to 30 seconds for my config

File: source/templates/classic/style.css [hunk 1]
```diff
@@ -891,7 +891,7 @@
     font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;
     font-weight: bold;
     font-size: 12px;
-    white-space: pre;
+    white-space: nowrap;
   }
   .added {
```
Comment (misnupur) [inline source/templates/classic/style.css:896]: 1000

**Review by misnupur — APPROVED** [review]
1000

**Pull request merged**